webui/react/src/pages/ExperimentList.tsx

```diff
@@ -80,13 +83,136 @@ const ExperimentList: React.FC = () => {
     },
   );
   const initSorter = storage.getWithDefault(STORAGE_SORTER_KEY, { ...defaultSorter });
-  const [ pagination, setPagination ] = useState<Pagination>({ limit: initLimit, offset: 0 });
-  const [ total, setTotal ] = useState(0);
+  const [ experiments, setExperiments ] = useState<ExperimentItem[]>();
   const [ filters, setFilters ] = useState<ExperimentFilters>(initFilters);
-  const [ sorter, setSorter ] = useState(initSorter);
+  const [ isUrlParsed, setIsUrlParsed ] = useState<boolean>(false);
+  const [ pagination, setPagination ] = useState<Pagination>({ limit: initLimit, offset: 0 });
   const [ search, setSearch ] = useState('');
-  const [ experiments, setExperiments ] = useState<ExperimentItem[]>();
   const [ selectedRowKeys, setSelectedRowKeys ] = useState<string[]>([]);
+  const [ sorter, setSorter ] = useState(initSorter);
+  const [ total, setTotal ] = useState(0);
+
+  /*
+   * When filters changes update the page URL.
+   */
+  useEffect(() => {
+    if (!isUrlParsed) return;
+
+    const searchParams = new URLSearchParams;
```

Comment:
I didn't know this was a thing where you can just create a new instance without the parentheses! (`new URLSearchParams` vs `new URLSearchParams()`) You learn something new everyday! :)